### ISSUE TITLE
Mark user owned probes in API responses

### ISSUE THREAD

**Issue by jimaek** [jsdelivr/globalping#541]
If the user is authenticated when making a new measurement we should clearly mark the probes he owns in the response.
Probably a system tag? This way the user could query his own probes directly as well.

Once the API supports it we can add markers to all of our tools.

Context https://github.com/jsdelivr/globalping-cli/issues/128

**Comment by MartinKolarik** [comment]
We discussed two options here.

1) The tag is something like `my-probes` - anonymous, not directly tied to the user account; it would only mean that the user who started the measurement owns the probe.
2) The tag is based on the username, e.g., `u-jimaek`. This means other people would be able to use these tags as well, and the probes wouldn't be anonymous. It would be opt-in/out.

I like the second option more because it solves a more generic problem. E.g. jsdelivr/www.jsdelivr.com#689 could use the same tags then.

**Comment by jimaek** [comment]
I dont think it applies to https://github.com/jsdelivr/www.jsdelivr.com/issues/689, the original text is outdated, the idea is to have a page per user, not per probe. So no opt-in per probe, its all or nothing, an account level setting.

So it sounds like the first option is better since option 2 only brings more issues

**Comment by MartinKolarik** [comment]
Option two assumes one account level setting as well. Basically, if you enable the page, we might as well tag all the probes (since they are listed under your name on that page). And at that point I don't think we need to add anything here.

**Comment by jimaek** [comment]
So a measurement response from any user or IP would always include the username in a tag and the tools would have to compare it to logged in user to highlight in the UI?

I guess that works. But it forces users to take extra steps and enable a feature they might not care about to simply get a small but useful UI feature.

It sounds like option 1 is needed for the UI highlighting and option 2 is a separate feature of public pages that makes your probes requestable by username

**Comment by MartinKolarik** [comment]
We need to make the toggle for public pages as easy to find as possible anyway to get people to enable the probes pages. Most likely, we'll be prompting them right after adopting a probe. So not really extra steps. Also:

> But it forces users to take extra steps and enable a feature they might not care about to simply get a small but useful UI feature.

The UI highlighting is very minor and mostly useless itself. I don't think it deserves its own mechanism when it isn't needed and you can even look at this as a benefit - if people actually want it, we'll get more probe pages too.

**Comment by jimaek** [comment]
The UI highlighting is what actually started this conversation and what people asked for. So its definitely not useless.

Everyone should have highlighting by default, its what they asked for and its useful. Forcing them to enable an unrelated feature is weird.

Your 2 options seem like 2 solutions for 2 different problems. Option 1 solves the UI highlighting and option 2 expands public pages and solves user targeting. So it sounds like we need to implement both for best UX.

**Comment by jimaek** [comment]
We should now update the CLI and web. Not sure if Slack makes sense though
